src/app/in-memory-data.service.ts

```diff
@@ -327,6 +327,89 @@ export class InMemoryDataService implements InMemoryDbService {
       'imageURL': 'https://avatars.githubusercontent.com/u/2410474?v=3'
     };
 
-    return {loginStatus, workitems, workitemtypes, user};
+    let linkCategories = {
```

Comment:
This is how a link category looks like. See this page: http://swagger.goa.design/?url=kwk%2Falmighty-core%2Fdesign#!/work-item-link-category/work_item_link_category_show

```json
{
  "data": {
    "attributes": {
      "description": "A work item link category that is meant only for work item link types goverened by the system alone.",
      "name": "system",
      "version": 0
    },
    "id": "6c5610be-30b2-4880-9fec-81e4f8e4fd76",
    "type": "workitemlinkcategories"
  }
}
```